ui/tests/acceptance/open-api-path-help-test.js

```diff
@@ -56,3 +57,52 @@ module('Acceptance | OpenAPI provides expected attributes enterprise', function
     }
   );
 });
+
+function secretEngineHelper(test, secretEngine) {
```

Comment:
Are these helpers that we could potentially import into other files (secret sync destinations, for example) to assert expected params?